src/fixtures/stubbed_logstash_index_pattern.js

```diff
@@ -1,21 +1,19 @@
-import _ from 'lodash';
 import TestUtilsStubIndexPatternProvider from 'test_utils/stub_index_pattern';
-import { IndexPatternsFieldTypesProvider } from 'ui/index_patterns/_field_types';
 import FixturesLogstashFieldsProvider from 'fixtures/logstash_fields';
+import { castEsToKbnFieldType } from '../utils';
 
 export default function stubbedLogstashIndexPatternService(Private) {
   const StubIndexPattern = Private(TestUtilsStubIndexPatternProvider);
-  const fieldTypes = Private(IndexPatternsFieldTypesProvider);
   const mockLogstashFields = Private(FixturesLogstashFieldsProvider);
 
-
   const fields = mockLogstashFields.map(function (field) {
-    field.displayName = field.name;
-    const type = fieldTypes.byName[field.type];
-    if (!type) throw new TypeError('unknown type ' + field.type);
-    if (!_.has(field, 'sortable')) field.sortable = type.sortable;
-    if (!_.has(field, 'filterable')) field.filterable = type.filterable;
-    return field;
+    const kbnType = castEsToKbnFieldType(field.type);
```

Comment:
This is always returning the unknown field type because `field.type` already has the kibana type name.